tests/Session/test_commands.py

```diff
@@ -41,39 +42,39 @@ def testCreateFreezer(self, mocked_input):
         # reset standout
         sys.stdout = sys.__stdout__
 
-        ## happy path: new freezer ##
-        mocked_input.side_effect = [profile_name, freezer_name]
+        ## happy path: new food storage ##
+        mocked_input.side_effect = [profile_name, foodstorage_name]
         self.commands.create_profile()
-        self.commands.create_freezer()
-        freezer = self.commands.profile.getFoodStorage(FREEZER, freezer_name)
-        self.assertIsNotNone(freezer)
-        self.assertTrue(isinstance(freezer, Freezer))
+        self.commands.create_foodStorage(freezer_name)
+        foodStorage = self.commands.profile.getFoodStorage(freezer_name, foodstorage_name)
+        self.assertIsNotNone(foodStorage)
+        self.assertTrue(isinstance(foodStorage, FoodStorage))
 
-        ## new freezer with existing name, invalid response, abort ##
-        # grab print output
-        capturedPrintOutput = io.StringIO()
-        sys.stdout = capturedPrintOutput
+        ## new food storage with existing name, invalid response, abort ##
+        # Grab print output from calling list()
+        capturedListOutput = io.StringIO()
+        sys.stdout = capturedListOutput
 
-        mocked_input.side_effect = [freezer_name, "x", "n"]
-        self.commands.create_freezer()
+        mocked_input.side_effect = [foodstorage_name, "x", "n"]
```

Comment:
I feel like "x" and "n" should be swapped for some constants

Reply:
Done!